<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_reBuildData">
         <property name="text">
          <string>重新生成数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reCount">
         <property name="text">
          <string>重新统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>图表主题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_chartTheme">
         <item>
          <property name="text">
           <string>Light</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BlueCerulean</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dark</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BrownSand</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BlueNcs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HighContrast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BlueIcy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Qt</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>图表动画效果</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_chartAnimation">
         <item>
          <property name="text">
           <string>NoAnimation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GridAxisAnimation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SeriesAnimations</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AllAnimations</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_quit">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>原始分数</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>学生人数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_studCount"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>人数统计</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <column>
              <property name="text">
               <string>分数段</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>数学</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>语文</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>英语</string>
              </property>
             </column>
             <item>
              <property name="text">
               <string>60分以下</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>60-69分</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>70-79分</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>80-89分</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>90分以上</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>BarChart</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>绘制柱状图</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>绘制水平柱状图</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="barChartView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>StackedChart</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_buildStackedBar">
              <property name="text">
               <string>绘制堆叠柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_buildHorizonStackedBar">
              <property name="text">
               <string>绘制水平堆叠柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QChartView" name="stackedChartView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>PercentBarChart</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton_percentBar">
              <property name="text">
               <string>绘制百分比柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_horizontalPercentBar">
              <property name="text">
               <string>绘制水平百分比柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QChartView" name="percentChartView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>PieChart</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>分析数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_course">
              <item>
               <property name="text">
                <string>数学</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>语文</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>英语</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_drawPieChart">
              <property name="text">
               <string>绘制饼图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>HoleSize</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_holeSizePieChart">
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.020000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>PieSize</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_pieSize">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.050000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_showLegend">
              <property name="text">
               <string>显示图例</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QChartView" name="pieChartView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
